### PR TITLE
Companion translate da

### PR DESCRIPTION
**Pull request by HThuren**
More translation inspired by ulfhedlund
Changed ChannelsPanel to Channels (seems more need to do that)

### CODE REVIEW

**Comment by ulfhedlund** [comment]
Glad you've had some use of my translations @HThuren :)
Not being a software developer myself I spent an awful lot of time figuring out how the translations work and what was missing. I used the French translations as inspiration and added from there. There was a lot missing and it seems a lot have changed/been moved over time so at least the Swedish translations still contain a lot of duplicates and old OpenTX stuff.

It would be awesome if a complete template or something similar could be created and also if there was a simple way of knowing what has been added recently (both radio firmware and Companion), so that the translations can be kept updated. 

Perhaps there are already tools or ways of managing translations in a (reasonably) simple way? I would expect so, to some degree.

**Comment by Eldenroot** [comment]
Lets use https://lokalise.com/?utm_source=google&utm_medium=cpc&utm_campaign=EMEA_LOK_Search_Competitor&utm_content=135185826381-607576676861&utm_term=crowdin&cq_src=google_ads&cq_cmp=17623500613&cq_term=crowdin&cq_plac=&cq_net=g&cq_plt=gp&gclid=Cj0KCQjwr4eYBhDrARIsANPywCgJmLbg0WNkIWkUjm0ExKiyFt3eLI54ZUFHr6eZ5F_N1k2IHjUYuxcaAt2cEALw_wcB

What do you think devs?

**Comment by pfeerick** [comment]
Yes, there will be a lot of stuff that is out of date in the translations in comparision to OpenTX 2.3 due to when they started working on OpenTX 2.4 (which then became the starting point for EdgeTX) and failure to keep 2.4 in sync with improvements that were being made in 2.3. 

As far as using a translation platform, that is a long-term goal. However, I would suggest we will be using https://crowdin.com, which has a free option as well as open source licensing options if we need more features than their free option offers. The free option of https://lokalise.com (mention of which is hidden in the FAQs) has a limit of 500 strings - which would be too limited when both companion and firmware strings are considered.

**Comment by HThuren** [comment]
Hi @ulfhedlund , of course I can use your translations, also to achieve the right approach to the final translation.
I has indeed used you translation, combined with the english.
To find strings, II simply place - in shell - me in the src map. From here I use "grep - r "word to find", sometimes look into the files (I know, this are development alike)

I agree with @pfeerick regarding tool, crowdin.com are easyy ttto use and has some help to search and look.

Before we go to crowdin.com I think at  tool to compare own language with the english files - what to expect, would be nice.

**Comment by HThuren** [comment]
hi @pfeerick, if is ok, please merge.

**Comment by HThuren** [comment]
Thank you

**Comment by HThuren** [comment]
Right, I was too fast to close PR :-)

**Comment by pfeerick** [comment]
*I wasn't going to say anything!* 😆 🙈 

All good... I had rebased it so only the unique changes were present and the mac companion build would finish happily this time (since that had been dealt with in the meantime), and then went to bed ;)
